apps/web/src/utils/csvUtils.tsx

```diff
@@ -32,6 +33,7 @@ import {
   getExperienceName,
 } from "~/utils/experienceUtils";
 import experienceMessages from "~/messages/experienceMessages";
+import { IndigenousCommunity } from "../api/generated";
```

Comment:
I am surprised you didn't use `@gc-digital-talent/graphql` 😆

Reply:
I am now! Auto import got me here 😛 

[prefer package over internal generated file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7376/commits/2a4f820077314d7128a47d96a2f6f36fa84adb57)